pkg/graphviz/traceflow.go

```diff
@@ -177,8 +177,14 @@ func getTraceflowMessage(o *opsv1alpha1.Observation) string {
 	if o.Component == opsv1alpha1.NetworkPolicy && len(o.NetworkPolicy) > 0 {
 		str += "\nNetpol: " + o.NetworkPolicy
 	}
+	if len(o.Pod) > 0 {
+		str += "\nTo: " + o.Pod
+	}
+	if o.Action != opsv1alpha1.Dropped && len(o.TranslatedDstIP) > 0 {
+		str += "\nTranslated Destination IP: " + o.TranslatedDstIP
```

Comment:
I am not sure the exact meaning of TranslatedDstIP here, maybe @gran-vmv can give some details for this field, so that we can enhance UI description for this.
Will it always be Pod IP? Or in the case where destination is service, it will be service IP instead?

Reply:
AFAIK, TranslatedDstIP is only set for service access and it's the destination pod ip.

Reply:
Got it, thanks Quan.